src/components/Book/CategoryView.tsx

```diff
@@ -7,16 +8,22 @@ type Props = {
 };
 export function BookCategoryView({ book }: Props) {
     const bestClass = book.bestInCategory ? ' best' : '';
+    const [active, setActive] = React.useState(true);
 
     const className = classNames({
         book: true,
         best: book.bestInCategory,
         'mb-3': true,
     });
 
+    const toggleBookCover = () => {
+        setActive(!active);
+        console.log('toggled');
+    };
+
     return (
-        <div className={className}>
-            <div className="book-cover">
+        <div onClick={toggleBookCover} className={className}>
```

Comment:
Can we use `classNames` instead? See example usage here: https://github.com/ContrarianMBA/www.contrarian.mba/blob/06f19a56f1c8082313c3e166d98317ff9f635380/src/components/Book/CategoryView.tsx#L11-L18

Let's use `classNames` and build up the styles in a separate variable if the `className` strings are ever complex.

Reply:
done